frontend/src/Components/ExercisesMain.js

```diff
@@ -10,39 +10,42 @@ import ExerciseLabel from './ExerciseLabel';
 
 const ExercisesMain = () => {
 
-    const [selected, setSelected] = useState({title: "select an exercise"})
+    const [labelsHeight, setLabelsHeight] = useState('85%')
+    const [selected, setSelected] = useState(null)
     const exercises = data.concat(data)
     //HTML
     return (
         <>
+            <Box h={'15%'}>
             <Heading as='h1' size='2xl' p={2}>
                 JS Exercises
             </Heading>
-            <Heading as='h2' size='lg' p={2}>
+            {selected ? (<Heading as='h2' size='lg' p={2}>
                 <i>{selected.title}</i>
-            </Heading>
+            </Heading>):<></>}
+            </Box>
 
-            <Box borderBottom='2px' borderColor='gray.200' h={580}>
+            {selected? (<Box borderBottom='2px' borderColor='gray.200' h={'60%'}>
                 <Flex>
                     <Spacer/>
                     {/* Left Block: */}
-                    <Exercise/>
+                    <Exercise selected={selected}/>
 
                     {/* Center Line: */}
                     <Spacer borderRight='2px' borderColor='gray.200'/>
                     <Spacer/>
 
                     {/* Right Block: */}
-                    <ExerciseExplanation/>
+                    <ExerciseExplanation selected={selected}/>
                     <Spacer/>
                 </Flex>
-            </Box>
+            </Box>):(<></>)}
             
-            <Box p={5} overflowY={"scroll"} h={"272"}>
+            <Box p={5} overflowY={"scroll"} h={labelsHeight}>
```

Comment:
adapted only for my screens, should do smth for phone version

monovalue leads to labels components being larger than body or too small, since explanations are not shown 
from the start, so it's 85% at the start then always 25%(each time an exercise is chosen the action of assigning height value repeats?)